packages/get-platform/src/__tests__/parseOpenSSLVersion.test.ts

```diff
@@ -72,6 +72,11 @@ describe('parseLibSSLVersion', () => {
       content: `/lib/libssl.so.3.1`,
       expect: '3.0.x',
     },
+    {
```

Comment:
Looks unrelated?

Reply:
It was a random missing test: https://github.com/prisma/prisma/pull/17389#discussion_r1073363280